exercises/debug-activity/README.md

```diff
@@ -12,6 +12,9 @@ Make your changes to the code in the `practice` subdirectory (look for
 the code). If you need a hint or want to verify your changes, look at 
 the complete version in the `solution` subdirectory.
 
+## Prerequisite: Ensure that the Temporal Service Is Running
```

Comment:
We generally note in our courses' root readmes that this is not necessary when using the live Gitpod environment, so we don't need to note it for each exercise.